### PR TITLE
Configurable database table name for storing the cart session id.

### PR DESCRIPTION
**Pull request by JuniorE**
Signed-off-by: JuniorE <janjunior.everaert@gmail.com>

### CODE REVIEW

**Comment by lukepolo** [comment]
@JuniorE can you look at my review?

**Comment by JuniorE** [comment]
> @JuniorE can you look at my review? 

Where can I see it? Sorry first time doing this.

**Comment by lukepolo** [comment]
No worries! 

If you click on "Files Changed"
image

You should see my comments and you can reply back to me through github.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor
 composer.lock
+.idea
```
[the remark this answers is not in the document]
Comment (lukepolo) [reply .gitignore]: but you can leave it doesn't hurt

File: src/Facades/LaraCart.php [hunk 2]
```diff
@@ -14,7 +14,7 @@ class LaraCart extends Facade
      *
      * @return string
      */
-    protected static function getFacadeAccessor()
```
[the remark this answers is not in the document]
Comment (JuniorE) [reply src/Facades/LaraCart.php:17]: No reason. PhpStorm changed that. I forgot to change it back to the original. My bad!

**Comment by lukepolo** [comment]
@JuniorE you can use 1.11.0 now~

**Pull request merged**